_docs/services/relational-database.md

```diff
@@ -117,38 +117,43 @@ In short, `cf bind-service` will provide a `DATABASE_URL` environment variable f
 
 The contents of the `DATABASE_URL` environment variable contain the credentials to access your database. Treat the contents of this and all other environment variables as sensitive.
 
-## Access the data in the database
+## Backups
+
+*Please note that these instructions will change in the future as we expand our service offerings and provide more options for customers.*
 
-There are currently two ways to access the database directly.
+For shared plans (`shared-psql` and `shared-mysql`), RDS does not back up your data.
```

Comment:
Per conversation with Carlo, can we revise this to "For shared plans (`shared-psql` and `shared-mysql`), cloud.gov and RDS does not back up your data. These are only intended for limited use development and testing instances."